doc/memos/rdm0.md

```diff
@@ -0,0 +1,153 @@
+represent the consensus within this community. The content of this document is
+licensed with a Creative Commons CC-BY-SA license.
+
+## Terminology
+
+This memo uses the [RFC2119](https://www.ietf.org/rfc/rfc2119.txt) terminology and the following acronyms:
+
+- RDM: RIOT Developer Memo
+- PR: git Pull-Request to RIOT's master branch on GitHub 
+- ACK: acknowledgement from a RIOT maintainer, concerning a pull-request to the 
+master branch of RIOT on GitHub
+
+# 1. Introduction
+
+In order to facilitate RIOT maintenance in the long term and at large scale, 
+additional documentation complementing code and usual code documentation is 
+needed. For instance, such memos are expected to describe and give explanations 
+about architectural design decisions, code structure etc. Other memos might
+also describe other aspects of RIOT activity, including, but not limited to,
+RIOT community processes, position with respect to some related external
+technical debates, pitfalls and best practices...
+
+
+This memo specifies a format for such memos and the process of discussing their 
+content before publication. Format and processes are somewhat inspired by 
+[IETF](http://ietf.org/) processes and documents such as Request For Comments 
+(RFC).
+
+# 2. RIOT Developers Memo Format
+
+RIOT Developer Memos (RDM) are written in English, using the Markdown (.md) 
+format, with less than 80 characters per line. Each memo has a unique 
+identification number indicated in its header, 
+e.g. this memo is RDM0.
+
+The structure of a memo MUST contain:
+
+1. a preamble as defined below,
+2. the content of the memo structured as needed,
+3. a postamble as defined below.
+
+The preamble of the memo should be structured as such:
+
+1. a header formatted as the header of this memo,
+2. a short abstract,
+3. a status for the memo stating its license and the category of the memo (e.g. 
+that it is a consensus document),
+4. optionally, a terminology section which recalls/disambiguates definition of 
+terms that are key to the memo.
+
+The core content of the memo is structured as deemed necessary. However, during 
+the review process, care must be taken to follow the below principles as much
+as possible:
+
+1. one memo = one problem/aspect,
+2. start by crisply explaining and motivating the one problem/aspect the memo 
+targets,
+3. memos are expected to be of normative and durable nature, hence: keep a memo 
+as concise as possible without impairing clarity, and factor out content that 
+would be likely to be updated too often.
+
+The postamble of the memo should be structured as such:
+
+1. (optional) acknowledgements section thanking contributors other than the 
+authors,
+2. a section with the main references that relate to the content, with a strong
+preference for permanent, self-contained references,
+3. the list of revisions so far and corresponding changelog,
+4. contact information to reach the author(s).
+
+
```

Comment:
Are these two new-lines intentionally (sorry, sometimes there are two newlines between sections, sometimes there aren't, sometimes between paragraphs; the pattern isn't clear to me)